engine/src/cubos/engine/renderer/viewport.cpp

```diff
@@ -0,0 +1,14 @@
+#include <cubos/core/ecs/component/reflection.hpp>
+#include <cubos/core/reflection/external/primitives.hpp>
+
+#include <cubos/engine/renderer/viewport.hpp>
+
+CUBOS_REFLECT_IMPL(cubos::engine::Viewport)
+{
+    return core::ecs::ComponentTypeBuilder<Viewport>("cubos::engine::Viewport")
+        .withField("positionX", &Viewport::positionX)
+        .withField("positionY", &Viewport::positionY)
+        .withField("sizeX", &Viewport::sizeX)
+        .withField("sizeY", &Viewport::sizeY)
+        .build();
+}
```

Comment:
```suggestion
}

```
Missing newline at EOF